src/openrct2/drawing/string.c

```diff
@@ -812,6 +812,12 @@ static void ttf_draw_string_raw_ttf(rct_drawpixelinfo *dpi, const utf8 *text, te
                             } else if (*dst < 10) {
                                 // For black backgrounds, hint using grey pixels.
                                 *dst = 13;
+                            } else if (colour == 153) {
```

Comment:
Is there no constant for this? I remember @marijnvdwerf replacing all instances.

Reply:
`interface/colour.h` has a few colour constants, but not for any of these. Please correct me if I'm overlooking any, of course.

Reply:
I think you need to add the appropriate PALETTE_INDEX_x entries in the enum, but I'd like to hear from @marijnvdwerf first.

Reply:
Either use `ColourMap`, or add a new `PALETTE_INDEX` entry.

Reply:
I've applied `ColourMapA` for now, where possible.  Suggestions for the other cases are welcome, too... I tried fiddling with the `BASE_COLOUR` macro as well, but I'm pressed for time and it wasn't quite working out yet, so I've left that out for now.